recipe/meta.yaml

```diff
@@ -395,10 +441,11 @@ outputs:
         - llvm =={{ version }}
         - zlib                               # [linux or win]
       run:
+        - ucrt                                          # [win]
         - {{ pin_compatible("libcxx", max_pin=None) }}  # [osx]
         - {{ pin_subpackage("clang-format", exact=True) }}
         - {{ pin_subpackage("libclang-cpp" + minor_aware_ext, max_pin="x.x") }}   # [unix]
-        - libclang {{ minor_aware_ext }}.*   # [unix]
+        - {{ pin_subpackage("libclang" + libclang_soversion, max_pin="x.x") }}
```

Comment:
Here I'm not 100% sure if `clang-tools` should depend on the versioned or unversioned `libclang`.

Reply:
unversioned is better.